co-authors-plus.php

```diff
@@ -1205,8 +1205,15 @@ public function ajax_suggest() {
 		}
 
 		foreach ( $authors as $author ) {
-			$avatar_url = get_avatar_url( $author->ID );
-			echo esc_html( $author->ID . ' | ' . $author->user_login . ' | ' . $author->display_name . ' | ' . $author->user_email . ' | ' . rawurldecode( $author->user_nicename ) ) . ' | ' . esc_url( $avatar_url ) . "\n";
+			printf(
+				"%s ∣ %s ∣ %s ∣ %s ∣ %s ∣ %s \n",
+				esc_html( $author->ID ),
+				esc_html( $author->user_login ),
+				esc_html( str_replace( '∣', '|', $author->display_name ) ),
```

Comment:
Hmm... I would explain the differences between the 2 characters in the first and second parameters here. 

It seems tricky and it does not work if the display name is using both characters like this`Jane | Doe ∣ do`. 

A workaround here is that using HTML entity for the 2nd parameter like this `&#x2223;`.

Reply:
I tried both using a JSON file and converting the pipe to its HTML entity. Both approaches were failing in this case, as the HTML entity will be seen as pipe and therefore also be handled as a delimiter and the JSON file will be visible in plain text to the user. Given that this is an edge case, I solved it by replacing the pipe sign with the divisor sign, which is not accessible on the keyboard by default.

That said, feel free to tinker with this PR. Maybe you'll find a way to improve the current PR. I simply don't want to spend too much time on this edge case.

Reply:
> Both approaches were failing in this case, as the HTML entity will be seen as pipe and therefore also be handled as a delimiter and the JSON file will be visible in plain text to the user

Yeah, that makes sense now. 
If using JSON, actually, the current code will need to be changed and handled with the JSON object rather than our own format. However, I've looked at the code and this may include so many changes, which is not necessary for this edge case.